video/clip.go

```diff
@@ -86,3 +98,24 @@ func ClipManifest(requestID string, manifest *m3u8.MediaPlaylist, startTime, end
 
 	return relevantSegments, nil
 }
+
+// Clips a segment by re-encoding so that I-frames are placed at the right intervals
+func ClipSegment(tsInputFile, tsOutputFile string, startTime, endTime float64) error {
+	start := formatTime(startTime)
+	end := formatTime(endTime)
+	err := ffmpeg.Input(tsInputFile).
+		Output(tsOutputFile, ffmpeg.KwArgs{"ss": start, "to": end,
+			"c:v":          "libx264",
+			"r":            "24",
```

Comment:
Could we have some comments explaining all these values please?

Reply:
Done